packages/deploy/deploy/700_land/land/742_setup_land_v4_royalty.ts

```diff
@@ -0,0 +1,44 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DEPLOY_TAGS} from '../../../hardhat.config';
+
+const ROYALTY_PERCENTAGE = 500;
+const func: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+): Promise<void> {
+  const {deployments, getNamedAccounts} = hre;
+  const {execute, read, catchUnknownSigner} = deployments;
+  const {royaltyManagerAdmin, commonRoyaltyReceiver} = await getNamedAccounts();
+  const Land = await deployments.get('Land');
+  const info = await read('Land', 'royaltyInfo', 1, 10000);
```

Comment:
Can you add a comment here, its not an obvious way to check royalties.

Alternatively can you consider calling getRoyaltyInfo on the RoyaltyManager to get the royalty?